packages/flutter/test/rendering/editable_test.dart

```diff
@@ -484,6 +484,7 @@ void main() {
       // This makes the scroll axis vertical.
       maxLines: 2,
       textSelectionDelegate: delegate,
+      // TODO(Piinks): Migrate these tests
```

Comment:
@chunhtai is there a migration guide for this? Can you advise how to migrate?

Reply:
you can create override userUpdateTextEditingValue in _FakeEditableTextState to listen to the selection changes